src/utils/log.js

```diff
@@ -32,17 +32,21 @@ const stepBanners = [
   },
   {
     name: 'Step 4',
-    description: 'Install, build and test the module, this may take a minute',
+    description: 'Initialize git',
   },
   {
     name: 'Step 5',
-    description: 'Initialize git',
+    description: 'Install, build and test the module, this may take a minute',
+  },
+  {
+    name: 'Step 6',
+    description: 'Initial Commit',
```

Comment:
Can we do a commit first that just has an empty README before one that has the generated content? This can make some branching issues easier later if they want to branch from the initial commit.

Reply:
Im not sure what the purpose of branching later from a commit with just the readme would be.

Reply:
I honestly don't recall what exactly the issue was, only that it was a major headache that would have been avoided with a regular initial commit rather than one that included the whole generated project.